calibration/proposed-tests/test_calibration_findings.py

```
@@ -0,0 +1,238 @@
+import tempfile
+import unittest
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Iterator
+
+ROOT = Path(__file__).resolve().parents[2] / "lean-code-gate-v3"
+GATE = ROOT / ".agent" / "lean" / "lean_code_gate.py"
+
+
+def run_gate(repo: Path, *args: str) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)